aiogram/fsm/storage/mongo.py

```diff
@@ -115,6 +115,23 @@ async def get_data(self, key: StorageKey) -> Dict[str, Any]:
             return {}
         return cast(Dict[str, Any], document["data"])
 
+    @overload
+    async def get_value(self, storage_key: StorageKey, dict_key: str) -> Optional[Any]: ...
+
+    @overload
+    async def get_value(self, storage_key: StorageKey, dict_key: str, default: Any) -> Any: ...
+
+    async def get_value(
+        self, storage_key: StorageKey, dict_key: str, default: Optional[Any] = None
```

Comment:
Actually what is the difference with the original implementation in the base method?

Is the same as get_data + get_value, so it is not needed to be overriden